presidio_evaluator/data_generator/faker_extensions/providers.py

```diff
@@ -7,15 +7,16 @@
 from faker.providers import BaseProvider
 from faker.providers.address.en_US import Provider as AddressProvider
 from faker.providers.phone_number.en_US import Provider as PhoneNumberProvider
+from presidio_evaluator.data_generator import raw_data
+
+_raw_data_dir = Path(raw_data.__path__[0])
```

Comment:
This is causing an issue with python 3.7: https://dev.azure.com/csedevil/Presidio/_build/results?buildId=14436&view=logs&j=00b6a206-eb91-585b-3d14-cf1a4d7b1970&t=9a9400f1-b707-5d54-e2fd-4fe1d8bc6dda

similar case: https://code.djangoproject.com/ticket/21862